02/List.js

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
-
+import {v4 as uuid} from 'uuid';
 import ListItem from './ListItem';
 
 function List(props) {
+
+    const {items} = props 
+
+    const listItems = items.map(item => <ListItem key={uuid()} item={item} /> )  
```

Comment:
`uuid()` powinno być generowane raz dla każdego elementu. Czyli np. w momencie dodawania nowego elementu.
W tym przypadku robisz to za każdym razem więc nie wiele nam daje `key`.